Test/scripts/ci-integration.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+Test/scripts/install_magento.sh
+cp Test/scripts/CouponCode.php ../magento
+cp Test/scripts/FreeShipping.php ../magento
+
+# install bolt plugin
+cd ..
+mkdir -p magento/app/code/Bolt/Boltpay
+cp -r project/. magento/app/code/Bolt/Boltpay/
+cd magento
+php bin/magento module:enable Bolt_Boltpay
+
+# # set config
+php bin/magento config:set payment/boltpay/active 1
+php bin/magento config:set payment/boltpay/api_key $BOLT_STAGING_MERCHANT_API_KEY
+php bin/magento config:set payment/boltpay/signing_secret $BOLT_STAGING_MERCHANT_SIGNING_SECRET
+php bin/magento config:set payment/boltpay/publishable_key_checkout $BOLT_STAGING_MERCHANT_PUBLISHABLE_KEY
+
+# install and run ngrok
+wget -O ngrok.zip https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
+unzip ngrok.zip
+./ngrok authtoken $NGROK_TOKEN
+./ngrok http 80 -hostname=m2-test.integrations.dev.bolt.me &
+sleep 10
+curl http://127.0.0.1:4040/api/tunnels
+NGROK_URL=$(curl http://127.0.0.1:4040/api/tunnels | grep  -oE '"public_url":"https://([^"]+)' | cut -c15-)/
+
+# TODO use proper store URL
+php bin/magento config:set web/unsecure/base_url "${NGROK_URL}"
+php bin/magento config:set web/secure/base_url "${NGROK_URL}"
+php bin/magento config:set web/unsecure/base_link_url "${NGROK_URL}"
+php bin/magento config:set web/secure/base_link_url "${NGROK_URL}"
+php CouponCode.php
+php FreeShipping.php
+
+
+php -dmemory_limit=5G bin/magento setup:upgrade
+
+php -dmemory_limit=5G bin/magento setup:di:compile
+
+php -dmemory_limit=5G bin/magento indexer:reindex
+
+php -dmemory_limit=5G bin/magento setup:static-content:deploy -f
+
+php bin/magento cache:flush
+INC_NUM=$((100*${CIRCLE_BUILD_NUM}))
+mysql -uroot -h 127.0.0.1 -e "USE magento2; ALTER TABLE quote AUTO_INCREMENT=${INC_NUM};"
+
+
+# tweak apache config
+echo "update apache config"
+sudo sh -c 'echo "<VirtualHost *:80>
```

Comment:
lets move these into static files or better yet as part of docker container build?